src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/NumberTableFilter.java

```diff
@@ -1,11 +1,9 @@
 package io.stargate.sgv2.jsonapi.service.operation.filters.table;
 
-import java.math.BigDecimal;
-
 /** Filter to use any JSON number against a table column. */
-public class NumberTableFilter extends NativeTypeTableFilter<BigDecimal> {
+public class NumberTableFilter extends NativeTypeTableFilter<Number> {
```

Comment:
Need to change now that we can get values in 3 distinct types (Long, BigInteger, BigDecimal).